SpokenChat/CSharp/SpokenChat.cs

```diff
@@ -4,10 +4,12 @@
 using Azure.AI.OpenAI;
 using Microsoft.CognitiveServices.Speech;
 using Microsoft.CognitiveServices.Speech.Audio;
+using System.Linq;
+using System.Text;
 
 // setup speech configuration 
 var speechConfig = SpeechConfig.FromSubscription(
-  Environment.GetEnvironmentVariable("AI_SERVICES_KEY"), "eastus2");
+    Environment.GetEnvironmentVariable("AI_SERVICES_KEY"), "eastus");
 
 // Get the text from the microphone
```

Comment:
I would clarify this is STT. Make it clear for learning how TTS, STT, and AOAI work together. Trying to follow the code I was also looking for "STT". :)
```suggestion
// Speech to text from the microphone
```